modules/local/benchmarking/location_plots/main.nf

```diff
@@ -0,0 +1,16 @@
+process LOCATION_PLOT {
+    label "process_single"
+
+    conda "bioconda::seaborn=0.11.2"
+    container 'uphl/seaborn'
+
+    input:
+        tuple val(id), path(bedfile1), path(bedfile2)
```

Comment:
What id is inserted here? Should this not be called `meta`? If so, you can add the `meta.id` as a [tag](https://www.nextflow.io/docs/latest/process.html#tag)